docs/modules/prompts/examples/output_parsing.ipynb

```diff
@@ -0,0 +1,161 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b492997a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.output_parsers.pydantic import PydanticOutputParser\n",
+    "from pydantic import BaseModel, Field"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "0ed75b4e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Response(BaseModel):\n",
+    "    answer: str = Field(description=\"answer to the user's question\")\n",
+    "    source: str = Field(description=\"source used to answer the user's question, should be a website.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ffb7fc57",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.prompts import PromptTemplate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "432ac44a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_parser = PydanticOutputParser(response_schema=Response)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "593cfc25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "prompt = PromptTemplate(\n",
+    "    template=\"answer the users question as best as possible.\\n{format_instructions}\\n{question}\",\n",
```

Comment:
I think it would be good to change the default here to ask the model to say "I don't know" when it doesn't know the answer to deal with questions containing incorrect information.

This is not related to the PR, but I think it might be a good safe default to introduce users of the library to.